contracts/StakingRewardsNotifier.sol

```diff
@@ -83,6 +83,6 @@ contract StakingRewardsNotifier is Ownable2Step, IStakingRewardsNotifier {
         delete mintedAmount;
```

Comment:
please remove this and remove the `mintedAmount` as we discussed

Reply:
Removed, moved comments saying why we have to keep `mintedAmount` in the function signature to the interface NatSpec.

Reply:
Also added usdc distribution to the notifier.